vmdb/product/toolbars/container_group_center_tb.yaml

```diff
@@ -16,9 +16,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"container_group")}'
       :title: 'Edit this #{ui_lookup(:table=>"container_group")}'
       :url: '/edit'
+      :enabled: 'false'
```

Comment:
@erinboyd why are we adding enabled: false here and few other places in other yaml files in this PR